app/src/main/java/org/xwiki/android/sync/auth/AuthenticatorActivity.kt

```diff
@@ -368,6 +369,7 @@ class AuthenticatorActivity : AccountAuthenticatorActivity() {
         val accountPassword = intent.getStringExtra(PARAM_USER_PASS)
         val accountServer = serverUrl
         val cookie = intent.getStringExtra(AccountManager.KEY_AUTHTOKEN)
+        val accessToken = intent.getStringExtra("access_token")
```

Comment:
`"access_token"` must be replaced to `const val`

Reply:
Okay.